lib/geo/config.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Geo.Config do
+  def json do
+    Application.fetch_env!(:geo, :json)
```

Comment:
I would rather try to fallback to something here instead of requiring it. For instance, if they have included a value there, use it, if not, Look to see if `Jason` is available, if it isn't available, look for `Poison`, if neither are available, then throw and error with a message letting users know what is needed

Reply:
@bryanjos from architecture point of view, I don't believe this is a good idea.

It seems to be handy because they do not have to add the configuration, sure I don't disagree there,

but

the amount of time they will have to touch the config for adding a key is too little compared with the afford and complexity that we add on the package (even they could copy the example from the readme and not do anything about it).

Today, is Jason or Poison the main JSON packages, but, if tomorrow a better package appear then people will start asking to add the checking for that one and so on.

The way is implemented right now embrace the onion architecture where you plug in your dependencies, at the moment we don't do that, we cross the boundaries of what I am trying to avoid on the package.

This is related to https://github.com/bryanjos/geo/issues/80#issue-294104146 where he wants the package to be aware of Poison details, which makes totally sense if Geo would depends of Poison.

Embrace people to be fine with configuring dependencies so the architecture of the package and the code do not suffer from that.

Reply:
I can agree to that, but would still like there to be a more friendly error message when it is not configured and that code is hit